contracts/modules/cover/Cover.sol

```diff
@@ -575,28 +526,8 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon {
 
   /* ========== Staking Pool creation ========== */
 
-  function createStakingPool(
```

Comment:
Moving this function entirely from Cover.sol means that PooledStaking needs to call the pool creator contract instead. Since the pool creator contract is not internal and stored as immutable it means that if there's the need to ever change something, we also need to redeploy Cover.sol and PooledStaking (at least until we decomission it ~3-4 months).

Reply:
We still need to proxy through Cover.sol because of the pool counter which will be stored in the Cover contract.

Reply:
yes